### ISSUE TITLE
Directory name "aux" is invalid on Windows.

### ISSUE THREAD

**Issue by mike-kaufman** [google/wuffs#33]
Hey folks -  commit cf6c5789736b19c1ed94c5e231e4db999e21c544 introduced a directory named "aux", which is invalid on windows. (see   [stack overflow](https://stackoverflow.com/questions/36225708/cannot-create-a-directory-named-aux-or-starting-by-aux-on-windows-8-1))

This results in failures during checkout on windows:

```
E:\chromium.wuffs>git checkout cf6c5789736b19c1ed94c5e231e4db999e21c544
fatal: cannot create directory at 'internal/cgen/aux': Invalid argument
```

**Comment by mike-kaufman** [comment]
See PR https://github.com/google/wuffs/pull/34 for possible fix

**Comment by nigeltao** [comment]
Fixed by PR #34.
